website/content/docs/k8s/helm.mdx

```diff
@@ -20,26 +20,22 @@ with Consul.
 
 Use these links to navigate to a particular top-level stanza.
 
-- [Helm Chart Reference](#helm-chart-reference)
-  - [Top-Level Stanzas](#top-level-stanzas)
-  - [All Values](#all-values)
-    - [global ((#h-global))](#global-h-global)
-    - [server ((#h-server))](#server-h-server)
-    - [externalServers ((#h-externalservers))](#externalservers-h-externalservers)
-    - [client ((#h-client))](#client-h-client)
-    - [dns ((#h-dns))](#dns-h-dns)
-    - [ui ((#h-ui))](#ui-h-ui)
-    - [syncCatalog ((#h-synccatalog))](#synccatalog-h-synccatalog)
-    - [connectInject ((#h-connectinject))](#connectinject-h-connectinject)
-    - [meshGateway ((#h-meshgateway))](#meshgateway-h-meshgateway)
-    - [ingressGateways ((#h-ingressgateways))](#ingressgateways-h-ingressgateways)
-    - [terminatingGateways ((#h-terminatinggateways))](#terminatinggateways-h-terminatinggateways)
-    - [apiGateway ((#h-apigateway))](#apigateway-h-apigateway)
-    - [webhookCertManager ((#h-webhookcertmanager))](#webhookcertmanager-h-webhookcertmanager)
-    - [prometheus ((#h-prometheus))](#prometheus-h-prometheus)
-    - [tests ((#h-tests))](#tests-h-tests)
-  - [Helm Chart Examples](#helm-chart-examples)
-  - [Customizing the Helm Chart](#customizing-the-helm-chart)
+- [`global`](#h-global)
```

Comment:
I am not sure... I just ran the generation...

Reply:
I see what happened @eddie-rowe looks like you introduce some more text into the generated page via this PR: https://github.com/hashicorp/consul/pull/17032/files. 

I assume safe to override your changes? We need to change the way we generate this file if you want those changes to persist as this page is actually maintained via golang code MDX generation.

Reply:
Interesting - I don't remember changing any of those higher level values.. maybe that was from a merge with another branch or resolving conflicts?

My changes for this particular page were to fix these broken links related to cluster peering:

https://github.com/hashicorp/consul/pull/17032/files#diff-d8efaedf4cf96374cd592339830873b0c609b0e0e5ddf30e74810058ab5142abR307-R1919

Reply:
@trujillo-adam @boruszak - do you have any perspective on these mentioned changes?

Reply:
It would be better to implement Eddie's changes in the source code  we generate this reference docs from, but it looks like we are already redirecting properly -- I think it's safe to overwrite his changes.